app/src/main/java/com/rosario/covid19/ui/HomeActivity.kt

```diff
@@ -86,9 +88,9 @@ class HomeActivity : AppCompatActivity(), View.OnClickListener {
                 calendar.set(Calendar.YEAR, year)
                 calendar.set(Calendar.MONTH, month)
                 calendar.set(Calendar.DAY_OF_MONTH, day)
-                tv_date.text = Util().userFormatDate(calendar)
+                dateTitleText.text = Util().userFormatDate(calendar)
```

Comment:
If you are going to make multiple calls to Utils class along the code, then I thinks it's better create a single instance and use that variable when you need to call the Utils class